flow-typed/src/ChorusClient.js

```diff
@@ -0,0 +1,12 @@
+interface ChorusClient {
+	open(url: string): Promise<Event>,
+	close(): void,
+	connect(): void,
+	publishStep(
+		pattern: string,
+		callback: (...args: Array<string>) => number | string | void,
+		technicalDescription?: string,
```

Comment:
Yes, that's right, that's optional. I think I basically tried to match what you documented here: https://github.com/nickebbutt/Chorus/wiki/WebSocketClientSpec and made parameters optional where they should be. Do let me know if I put some as optional when in fact they shouldn't be. Thanks!